src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToLatexFormatterTest.java

```diff
@@ -45,7 +45,7 @@ public void testHTML() {
         assertEquals("{\\\"{a}}", formatter.format("&auml;"));
         assertEquals("{\\\"{a}}", formatter.format("&#228;"));
         assertEquals("{\\\"{a}}", formatter.format("&#xe4;"));
-        assertEquals("{$\\Epsilon$}", formatter.format("&Epsilon;"));
+        assertEquals("{{$\\Epsilon$}}", formatter.format("&Epsilon;"));
```

Comment:
Well, ä is correctly written as  {\"{a}} which as a Java String becomes
that.

\Epsilon is only valid in math mode and for the \Epsilon to stay a capital
\Epsilon double {} are needed in the title field, see #405.